cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -60,6 +60,9 @@
   /** name of the primary key column */
   private static final String PK_COLUMN_NAME = "id";
 
+  private final Pattern uuidPattern = Pattern.compile("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$");
+  private final Pattern tableNamePattern = Pattern.compile("^[0-9a-zA-Z_-]+\\.[0-9a-zA-Z_-]+$");
```

Comment:
The tableNamePattern matches 1.5 but this is not a valid table name.
Minus is an invalid character for a table name.
See https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS for the spec.
My suggestion is "^[a-zA-Z_][0-9a-zA-Z_]*\\.[0-9a-zA-Z_]+$"